lib/workload/stateless/stacks/bs-runs-upload-manager/deploy/lib/stacks/bs_runs_upload_manager_stack.ts

```diff
@@ -0,0 +1,70 @@
+import * as cdk from 'aws-cdk-lib';
+import { Construct } from 'constructs';
+import { BsRunsUploadManagerConstruct } from '../constructs/bs_runs_upload_manager_stack';
+import { LambdaLayerConstruct } from '../constructs/lambda_layer';
+import * as secretsManager from 'aws-cdk-lib/aws-secretsmanager';
+import * as events from 'aws-cdk-lib/aws-events';
+import * as ssm from 'aws-cdk-lib/aws-ssm';
+// import * as sqs from 'aws-cdk-lib/aws-sqs';
+
+export interface BsRunsUploadManagerConfig {
+  // Define construct properties here
+  ica_token_secret_id: string; // IcaSecretsPortal
+  portal_token_secret_id: string; // orcabus/token-service-jwt
+  basespace_token_secret_id: string; // /manual/BaseSpaceAccessTokenSecret
+  gds_system_files_path: string; // gds://development/primary_data/temp/bs_runs_upload_tes/
+  eventbus_name: string; // /umccr/orcabus/stateful/eventbridge
+}
+
+export type BsRunsUploadManagerStackProps = BsRunsUploadManagerConfig & cdk.StackProps;
+
+export class BsRunsUploadManagerStack extends cdk.Stack {
+  constructor(scope: Construct, id: string, props: BsRunsUploadManagerStackProps) {
+    super(scope, id, props);
+
+    // Set lambda layer arn object
+    const lambda_layer_obj = new LambdaLayerConstruct(this, 'lambda_layer', {
+      layer_directory: __dirname + '/../../../layers',
```

Comment:
Not sure this string concat might work... We are doing `path.join()` like so:

```
layer_directory: path.join(__dirname, '/../../../layers'),
```

And,

```
import path from 'path';
```

And, this has `devDependencies` the following; of which had it in outer package.json.
```
"@types/node": "^20.4.0"
```

Reply:
But, if you find concat work then that's fine too.

Reply:
Yep, fixed in b4041ad7905c37aaccd6af7fb0f9180cb1910213